fundingmanager.go

```diff
@@ -1353,6 +1417,19 @@ func (f *fundingManager) handleFundingSigned(fmsg *fundingSignedMsg) {
 				},
 			},
 		}
+
+		f.deleteReservationCtx(peerKey, pendingChanID)
+
+		err = f.annAfterSixConfs(completeChan, shortChanID)
+		if err != nil {
+			fndgLog.Errorf("failed sending channel announcement: %v",
+				err)
+			return
+		}
+		fndgLog.Debugf("Channel with ShortChanID %v announced",
```

Comment:
I think this logging statement should be moved into the method above. As otherwise, we get conflicting logs like this (the channel was set to private):
```
2017-12-13 16:08:45.514 [DBG] FNDG: Channel with ShortChanID 1830686860312576 added to router graph
2017-12-13 16:08:45.514 [DBG] PEER: Received FundingLocked(chan_id=32d05b577966bb3280a58c8f2d7cbd2e0a916c4bdce67d8c024ffb693ccf2d88, next_point=033201ae22cbf6fa8cb07d0f2187ab764f33d8378bf830c1bc9b6e21063f5b7be6) from 127.0.0.1:10019
2017-12-13 16:08:45.516 [TRC] PEER: readMessage from 127.0.0.1:10019: (*lnwire.FundingLocked)(0xc4205981e0)({
 ChanID: (lnwire.ChannelID) (len=32 cap=32) 32d05b577966bb3280a58c8f2d7cbd2e0a916c4bdce67d8c024ffb693ccf2d88,
 NextPerCommitmentPoint: (*btcec.PublicKey)(0xc420174840)({
  Curve: (elliptic.Curve) <nil>,
  X: (*big.Int)(0xc420174860)(22618611118210622400841714471844468386370929431841625480347924787655014579174),
  Y: (*big.Int)(0xc420174880)(5784334520023794081739208847968927079355214281329805474126131064251764117719)
 })
})

2017-12-13 16:08:45.516 [DBG] FNDG: Received FundingLocked for ChannelID(32d05b577966bb3280a58c8f2d7cbd2e0a916c4bdce67d8c024ffb693ccf2d88) from peer 02cc27b8ef708b13a6832f1c4f3d8d1207851cda53f5be041f5b1e2ec1afeb755c
2017-12-13 16:08:45.515 [TRC] PEER: writeMessage to 127.0.0.1:10019: (*lnwire.ChannelUpdate)(0xc420082cd0)({
 Signature: (*btcec.Signature)(0xc42026a0e0)({
  R: (*big.Int)(0xc42023c1c0)(42098235645066207283475299495848826319528238987901551247437816008865568379280),
  S: (*big.Int)(0xc42023c220)(36309963118044616166236141227015067109229028731851401167576737900038110028029)
 }),
 ChainHash: (chainhash.Hash) (len=32 cap=32) 683e86bd5c6d110d91b94b97137ba6bfe02dbbdb8e3dff722a669b5d69d77af6,
 ShortChannelID: (lnwire.ShortChannelID) {
  BlockHeight: (uint32) 1665,
  TxIndex: (uint32) 1,
  TxPosition: (uint16) 0
 },
 Timestamp: (uint32) 1513210125,
 Flags: (lnwire.ChanUpdateFlag) 1,
 TimeLockDelta: (uint16) 144,
 HtlcMinimumMsat: (lnwire.MilliSatoshi) 0 mSAT,
 BaseFee: (uint32) 1000,
 FeeRate: (uint32) 1
})

2017-12-13 16:08:45.516 [DBG] FNDG: Will not announce private channel 1830686860312576.
```